src/api_client.py

```
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import requests
+from src import config
+
+def send_vacante(data_vacante):
```

Ruff (0.12.2)

Missing return type annotation for public function `send_vacante`

Add return type annotation: `None`

(ANN201)